data/hai/unfettered side missions.txt

```diff
@@ -0,0 +1,484 @@
+					goto scare
+			`	"I guess we'll find out." Says a third.`
+			label scare
+			`	The leader gets closer and addresses you in a more serious tone, "You should be aware, <last>, that this is not a friendly competition between allies. As far as I am concerned, you are an enemy. You should see it more as a fight for you to truly earn this ship, and more, but make no mistakes: it has only two rules: nobody else than us may interfere, and being disabled results in losing the fight. And do not even think about bringing more than this Ladybug as a flagship or we will not bother showing up!"`
+			# this is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles
+			# (but in reality with disabling a lot of ships it goes faster than that)
+			`	They suddenly look at you intensively, straight in the eyes, "We sometimes bend the rules for our friends, but that is not your case," they say with a sudden change of tone, before leaving.`
+				to display
+					"reputation: Hai (Unfettered)" < -2000
+			`	The engineer comes back, looking a bit impatient, "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships to get the initiative. The engineer utters a word to you once they are far enough away, "I want this done by the end of the day, do not try my patience. You are lucky I offered this deal in the first place, make sure I do not regret it. Now, quick, get to your ship!" You see the contestants have already begun the launching procedures, and launch as soon as you can.`
+			`	(Remember to park your other ships and make sure your flagship is the Ancient Ladybug.)`
+				accept
+	on decline
+		"reputation: Hai (Unfettered)" >?= -1000
+	on disabled
+		fail "Hai Legacy 3"
+		dialog `The enemy ships hail you, "You have failed the challenge. We will take the ship from you when we see fit, I heavily suggest you land on Darkcloak so that we take it from you without trouble - we can even leave you with the outfits. Do not try to keep it from us!"`
+		set "unfettered: undeserved ladybug"
+	on fail
+		"reputation: Hai (Unfettered)" >?= -1000
+		dialog `The Unfettered hail you, "You failed to complete the duel properly, and as such, are no longer authorized to land on our worlds. You will not enjoy unwarranted respect any longer!"`
+	npc disable
+		government "Hai (Unfettered Challenger)"
+		to spawn
+			"total ships" == 1
+			has "flagship model: Ancient Ladybug"
+		to despawn
+			"total ships" != 1
+		personality unconstrained launching plunders staying target heroic disables
+		ship "Lightning Bug (Ionic)" "Erazer"
+		ship "Lightning Bug" "Thunder"
+		ship "Lightning Bug (Unfettered Shipyards)" "Lighting"
+		ship "Lightning Bug (Pulse)" "Struck"
+		conversation
+			`You receive a message from the remaining Unfettered ships: "We surrender." You remember you were asked to land on the same day in order to claim your reward of permanent access to Unfettered space.`
```

Comment:
```suggestion
			`You receive a message from the remaining Unfettered ships: "We surrender. If you land now, you will receive your reward."`
```
I think having the player "remember" something is against the "don't assume stuff about the player".

Reply:
~~alright, you may have one day amnesia, got it~~